draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -17,32 +17,13 @@ export type TooltipProps = {
    * @deprecated
    */
   inline?: boolean
-  /**
-   * Unfortunately, the content needed to be wrapped in a div. This can sometimes
-   * break the css layout. To get around this, we allow you to specify the css
-   * display value directly. If you need to need to modify more values, feel free
-   * to use the `classNameOverride` prop, but avoid it if you can.
-   */
   display?: "block" | "inline" | "inline-block" | "flex" | "inline-flex"
```

Comment:
Might be a bit heavy handed, but I think this kind of description isn't really helpful within the IDE and much better shown together with examples. I had no idea what these meant until I did the example.

Reply:
Tbh a bit hard to describe the use case for this...